2018-10-11 19:02:13.169 -03:00 [Information] Applying migrations for Identity...
2018-10-11 19:02:16.396 -03:00 [Information] Migrations applied.
2018-10-11 19:02:16.397 -03:00 [Information] Filling database with system users and roles...
2018-10-11 19:02:17.108 -03:00 [Information] Database filled.
2018-10-11 19:12:25.392 -03:00 [Error] An internal error occurred [LogId:ba586ed0-c608-4920-a941-ec83ceb34a5b].
System.InvalidOperationException: ah vsf vai.
   at WebApplication1.Controllers.ValuesController.Get(Int32 id) in C:\_ivanborges\github\furiza-aspnetcore\src\WebApplication1\Controllers\ValuesController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-11 20:03:24.613 -03:00 [Information] Applying migrations for Identity...
2018-10-11 20:03:27.253 -03:00 [Information] Migrations applied.
2018-10-11 20:03:27.254 -03:00 [Information] Filling database with system users and roles...
2018-10-11 20:03:28.048 -03:00 [Information] Database filled.
2018-10-11 20:04:15.454 -03:00 [Error] An internal error occurred [LogId:7d39b6c3-6928-4a70-b5f4-511e9b95e93a].
System.InvalidOperationException: ah vsf vai.
   at WebApplication1.Controllers.ValuesController.Get(Int32 id) in C:\_ivanborges\github\furiza-aspnetcore\src\WebApplication1\Controllers\ValuesController.cs:line 45
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
2018-10-11 20:04:18.152 -03:00 [Information] O CPF fornecido está em branco, é inválido ou está em um formato inválido.
Furiza.Base.Core.Domain.BR.Exceptions.CpfInvalidoException: O CPF fornecido está em branco, é inválido ou está em um formato inválido.
   at WebApplication1.Controllers.ValuesController.Get(Int32 id) in C:\_ivanborges\github\furiza-aspnetcore\src\WebApplication1\Controllers\ValuesController.cs:line 48
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Furiza.AspNetCore.ExceptionHandling.ExceptionMiddleware.Invoke(HttpContext context) in C:\_ivanborges\github\furiza-aspnetcore\src\Furiza.AspNetCore.ExceptionHandling\ExceptionMiddleware.cs:line 28
